content/en/docs/tasks/observability/metrics/customize-metrics/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Customizing Istio Metrics
+description: This task shows you how to customize the Istio metrics.
```

Comment:
Should this still be marked as Experimental?

Reply:
Got answer: Alpha